tests/sentinel/tests/16-config-set-config-get.tcl

```diff
@@ -56,3 +56,11 @@ test "SENTINEL CONFIG SET, wrong number of arguments" {
         fail "Expected to return Missing argument error"
     }
 }
+
+test "Undo SENTINEL CONFIG SET changes" {
+    foreach_sentinel_id id {
+        assert_equal {OK} [S $id SENTINEL CONFIG SET resolve-hostnames no announce-port 0]
+    }
+    assert_match {*no*0*} [S 1 SENTINEL CONFIG GET resolve-hostnames announce-port]
+    assert_match {announce-port 0} [S 1 SENTINEL CONFIG GET announce-port]
```

Comment:
It turns out that the sentinel test framework does not run tests completely independent. Configuration changes like the ones tested in this test are persisted and used in the following tests. 

I considered three options:
1. Change the test framework: While this might be a good idea, I did not want to do this in this PR
2. Undo the config changes at the end of the "config-set-config-get" test.
3. Keep "config-set-config-get" at the end and insert the new tests "manual-coordinated" and "stuck-failover" in the middle, e.g. after "05-manual"

I took option 2 and that's why this code is here. But I could also try option 3 and rename the test files.

Reply:
I deleted line 65 for the time being.